### PR TITLE
Task5

### PR DESCRIPTION
**Pull request by gadjevb**
Task5 is ready for review!
https://gadjevb.github.io/

### CODE REVIEW

File: task3 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+			Address:<br>
+			<input type="text" name="address">
+			<br>
+			Password:<br>
+			<input type="password" name="password" id="pass">
+			<br>
+			Confirm password:<br>
+			<input type="password" name="confirmPassword" id="conPass">
+			<br>
+			<input type="submit" value="Submit" onclick="return validateForm()">
```
Comment (freekman) [inline task3]: remove the submit button and make every field to be validted on lose focus

File: task5.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        return false;
+    }
+	if (address == null || address == "" || address.length > 100) {
+        alert("Address must be filled out, length must be hundred characters or less!");
+        return false;
+    }
+	if (password == null || password == "" || password.length > 18) {
+        alert("Password must be filled out, up to 18 character are allowed!");
+        return false;
+    }else{
```
Comment (freekman) [inline task5.js]: too complex

File: task5.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	var password = document.getElementById("pass").value;
+	var passwordCon = document.getElementById("conPass").value;
+	var i;
+	var temp;
+	
+    if (firstName == null || firstName == "" || firstName.length > 15) {
+        alert("First name must be filled out, 1 to 15 symbols allowed!");
+        return false;
+    }
+	if (lastName == null || lastName == "" || lastName.length > 15) {
```
Comment (freekman) [inline task5.js]: code duplication

File: task5.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        alert("Last name must be filled out, 1 to 15 symbols allowed!");
+        return false;
+    }
+	if (!egn.match(/^\d+$/) || egn.length != 10) {
+        alert("EGN must be filled out with exactly 10 numbers!");
+        return false;
+    }
+	if (!age.match(/^\d+$/) || age < 18 || age > 118) {
```
Comment (freekman) [inline task5.js]: .

File: task5.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+	if (value.length < 1 || value.length > 15) {
+        alert(name + " must be filled out, 1 to 15 symbols allowed!");
+    }
+}
+
+function validateEGN(egn) {
+	var value = document.forms["Registration"][egn].value;
+
+	if (!value.match(/^\d+$/) || value.length != 10) {
```
Comment (freekman) [inline task5.js]: value.length != 10 can abe removed

**Comment by gadjevb** [comment]
Done, now it's just with regex!

**Pull request closed without merging**